index.js

```diff
@@ -631,6 +633,21 @@ const publicApiProxy = new Proxy(publicApi, {
                     process.exit(1); // eslint-disable-line
                 }
             };
+        } else if (typeof target[prop] === 'undefined') {
```

Comment:
nit pick: since we return in the first `if`, this doesn't need to be a nested `else if`, it can just be a fresh, un-nested `if`.